_Sidebar.md

```diff
@@ -96,6 +96,7 @@
     * [[How to handle merging of change lists for exploration properties|Guide-to-handle-merging-of-change-lists-for-exploration-properties]]
     * [[Instructions for editing roles or actions|Instructions-for-editing-roles-or-actions]]
     * [[Protocol buffers|Protocol-buffers]]
+    * [[Math Interactions Overview | Math-Interactions-Overview]]
```

Comment:
This doesn't seem like a backend thing? It feels more fullstack, maybe put it as a sub-point of "Interactions" below.

Reply:
Done.

Reply:
No, I don't mean that. Lower down, in the sidebar, there is a section for "Interactions", under "Extension frameworks". Please add a sub-point there instead, since this is a subclass of that.

Reply:
Done.